code/update.lua

```diff
@@ -1,7 +1,9 @@
 print("Installing cc-code...")
 
-fs.delete("code.lua")
-fs.delete("code")
+if fs.exists("code/") and fs.exists("code.lua") then
+  fs.move("code/","codeOld/")
+  fs.move("code.lua","code.lua.old")
```

Comment:
```suggestion
  fs.move("/code", "/code_old/code")
  fs.move("/code.lua", "/code_old/code.lua")
```

I'd rather have the entire backup in one directory. Also there's a chance `code_old` already exists, but I'll add some better handling for that myself at some point. This should be good enough for now.